app/components/primer/alpha/select_panel.rb

```diff
@@ -429,6 +409,11 @@ def initialize(
         @dynamic_aria_label_prefix = dynamic_aria_label_prefix
         @loading_label = loading_label
         @loading_description_id = nil
+
+        @form_builder = form_arguments[:builder]
+        @default_value = form_arguments[:default_value]  
+        @input_name = form_arguments[:name]
```

Comment:
Extract all form_arguments and add an optional argument `default_value` so that remote and eventually_local strategies can pass in a pre-set value.